### ISSUE TITLE
Unknown config option ignore was found

### ISSUE THREAD

**Issue by steriley** [vue-styleguidist/vue-styleguidist#191]
By [following the documentation](https://github.com/vue-styleguidist/vue-styleguidist/blob/master/docs/Cookbook.md#how-to-exclude-some-components-from-style-guide) on the example of how to ignore a component it requests you to add the following into `styleguide.config.js`

`ignore: ['**/*.spec.vue', '**/components/Button.vue']`

However when doing so the following error comes back:

```
true 'StyleguidistError'
Unknown config option ignore was found, the value is:
[
  '**/*.spec.vue',
  '**/components/Button.vue'
]
undefined
error Command failed with exit code 1.
```

This leads me to believe there is a bug with ignore at the moment or the documentation is out of date for this option?

**Comment by viljamis** [comment]
Getting this same issue while trying to update Vue Design System dependencies.

**Comment by viljamis** [comment]
Investigated a bit further and I’m not sure anymore if this is related to changes in Vue Styleguidist. Earlier today everything was working fine when doing an NPM install, but now, even if installing a year old version, I get this error on all repositories. Weird.

**Comment by viljamis** [comment]
@rafaesc eventually it looks like this issue is related to the latest version (1.8.4) + vue & vue-template-compiler. If using vue-styleguidist v1.8.3 and forcing NPM to use the previous vue (2.5.16) and vue-template-compiler (2.5.17) everything works just fine. But if updating either one things break completely.

<img width="300" alt="screenshot 2018-09-06 16 28 03" src="https://user-images.githubusercontent.com/671836/45160398-080be780-b1f2-11e8-837b-900ffba53869.png">

**Comment by rafaesc** [comment]
Fixed on version 1.8.5

**Comment by viljamis** [comment]
@rafaesc Awesome! Thank you!
